{
  "_annotation": {
    "cve_id": "CVE-2024-39887",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-39887.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:apache:superset:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "18D907E1-873E-56D9-9E00-40E48C9C86D9",
                "versionEndExcluding": "4.0.2",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}